Web Experiences/Blueprints/alert-bar/alertBarScroll.js

```diff
@@ -32,12 +39,22 @@ function sendDataToBoxever(eventType) {
         "interactionID":"OOB_EXP",
 	    "interactionName": "ALERT_BAR_SCROLL"
     };
-    Boxever.eventCreate(eventToSent, function (data) { }, 'json');
+    Boxever.eventCreate(eventToSent, (data)=>{ }, 'json');
```

Comment:
I think this syntax is not supported on IE11. As far as I can tell, everything else here is, so it would be a shame to lose out on IE11 support just so we can use this syntax.

Reply:
I changed all the arrow functions back across all those 4 Experiences